examples/Bonadrone/SBUSLevel/SBUSLevel.ino

```diff
@@ -48,31 +48,47 @@ hf::SBUS_Receiver rc = hf::SBUS_Receiver(CHANNEL_MAP, SERIAL_SBUS, &SBUS_SERIAL)
 
 hf::MixerQuadX mixer;
 
+// Rate Constants
```

Comment:
These are constants for Mosquito 90. When we launch the release, we may provide the parametrized sketch along with param files to configure properly either the Mosquito 90 and the Mosquito 150.

Reply:
I will put them in a separate file so that we can keep track of them

Reply:
https://github.com/BonaDrone/documentation/blob/master/controllers/M90-constants.md